src/controller/index.ts

```diff
@@ -412,6 +419,24 @@ export default class SimulariumController {
         return this.playBackFile;
     }
 
+    public checkServerHealth(
+        handler: () => void,
+        netConnectionConfig: NetConnectionParams
+    ): void {
+        if (
+            !this.simulator ||
+            !(this.simulator instanceof RemoteSimulator) ||
+            !this.simulator.socketIsValid()
+        ) {
+            // Only configure network if we aren't already connected to the remote server
+            this.configureNetwork(netConnectionConfig);
```

Comment:
It feels a bit funny to have to pass the config in every time we make a check, it looks redundant on the front-end, but it does ensure we aren't querying an un-configured controller that won't respond.

Reply:
Yeah it does sort of make sense that you might want to check health before a connection was made.